db-scripts/src/main/resources/migration.sql

```diff
@@ -720,3 +720,22 @@ CASE
     ELSE 1
 END;
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.11.0";
+
+##version 2.11.1
+REPLACE INTO sample_list(STABLE_ID, CATEGORY, CANCER_STUDY_ID, NAME, DESCRIPTION)
+    SELECT DISTINCT CONCAT(CANCER_STUDY_IDENTIFIER, '_fusion'), 'all_cases_with_mutation_data', `cancer_study`.CANCER_STUDY_ID, 'Samples with fusion data','Samples with fusion data'
+        FROM `mutation_event`
+            JOIN `mutation` ON `mutation`.MUTATION_EVENT_ID = `mutation_event`.MUTATION_EVENT_ID
+            JOIN `genetic_profile` ON `genetic_profile`.GENETIC_PROFILE_ID = `mutation`.GENETIC_PROFILE_ID
+            JOIN `cancer_study` ON `cancer_study`.CANCER_STUDY_ID = `genetic_profile`.CANCER_STUDY_ID
+        WHERE MUTATION_TYPE = 'fusion';
+REPLACE INTO sample_list_list(list_id, sample_id)
+    SELECT DISTINCT list_id, SAMPLE_ID
+        FROM `mutation_event`
+            JOIN `mutation` ON `mutation`.MUTATION_EVENT_ID = `mutation_event`.MUTATION_EVENT_ID
+            JOIN `genetic_profile` ON `genetic_profile`.GENETIC_PROFILE_ID = `mutation`.GENETIC_PROFILE_ID
+            JOIN `cancer_study` ON `cancer_study`.CANCER_STUDY_ID = `genetic_profile`.CANCER_STUDY_ID
+            JOIN `sample_list` ON `sample_list`.CANCER_STUDY_ID = `cancer_study`.CANCER_STUDY_ID AND `sample_list`.STABLE_ID LIKE '%fusion%'
+        WHERE MUTATION_TYPE = 'fusion';
```

Comment:
My understanding of this code is that a default _fusion sample list will be created which is the list of samples that have fusion data. There is a caveat: some profiled samples may be fusion negative. For example, many MSK-IMPACT samples do not have fusions but they were all profiled.

This might be fine as the default one for migration, but we should generate proper _fusion case list for our studies.

Alternatively, I am wondering if we should just copy _sequenced to _fusion when there are fusion data?

@n1zea144 @sheridancbio thoughts?

Reply:
@n1zea144 @SRodenburg , any inputs on this? thanks!

Reply:
Hi @jjgao Rob and I were talking about this too.  Using MSK-IMPACT-ARCHER as an example, I'm not sure if its 100% accurate to assume that if a sample was sequenced for mutation calling, that it was also analyzed for fusions (although it is true that with impact, there are some probes looking for canonical fusions).  However, it seems excessive to have a separate list for cases that were checked for SVs, and then another case list for samples in which SVs were found.

Reply:
thanks, @n1zea144 ! 
but how do we know if there are fusions or not at the importing stage to decide if we should carry the sequenced sample list over to fusions?

Reply:
here is my solution:
1. use _fusion case list to add fusion positive samples to the sample list table.
2. SampleServiceImpl populates fusionSampleMap using fusion case list, but uses sequencedSampleIdsMap to actually mark samples **_fusion present_** if fusionSampleMap is not empty.

what do you think, @jjgao , @n1zea144 , @sheridancbio?